lua/wikis/commons/Widget/Infobox/Cell.lua

```diff
@@ -46,7 +46,10 @@ function Cell:render()
 
 	local options = self.props.options
 
-	local mappedChildren = Array.map(self.props.children, function(child)
+	-- map children to strings so we can add widgets (which potentially render empty) without getting empty cells displayed
+	local children = Array.map(self.props.children, tostring)
```

Comment:
empty, as in empty string, or nil?

Reply:
technically it checks for the first children being empty via `Logic.isEmpty` so it would catch both empty string and nil

but i think using widgetUtil.collect might be a better solution, so will adjust to that